solidity/contracts/converter/BancorConverter.sol

```diff
@@ -479,7 +523,7 @@ contract BancorConverter is IBancorConverter, SmartTokenController, ContractRegi
       * @return amount of tokens received (in units of the target token)
     */
     function convertInternal(IERC20Token _fromToken, IERC20Token _toToken, uint256 _amount, uint256 _minReturn)
-        internal
+        public

```

Comment:
@yudilevi : Good catch! It probably happened when I changed the `getXXXReturn` functions from `public` to `internal`, and then back to `public` in order to avoid breaking the tests (so when I changed them back to `public`, I must have changed that one to `public` too).
Fixed.